app/models/miq_worker.rb

```diff
@@ -504,7 +503,7 @@ def clean_active_messages
   end
 
   private def set_system_uid
-    self.system_uid = unit_name if systemd_worker?
+    self.system_uid = unit_name unless containerized_worker?
```

Comment:
`unit_name` isn't ideal for Process mode since it is primarily a systemd concept, but it is just the worker.class.underscore plus the guid which also applies to Processes